webpack/assets/javascripts/react_app/components/HostDetails/Audits/index.js

```diff
@@ -2,6 +2,8 @@ import PropTypes from 'prop-types';
 import React, { useEffect, useState } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
 import Skeleton from 'react-loading-skeleton';
+import ArrowIcon from '@patternfly/react-icons/dist/js/icons/arrow-icon';
```

Comment:
Is it wise to import from internals of `@patternfly`? Aren't they exporting it somehow?

Reply:
It's a public extension - 
https://patternfly-react.surge.sh/extensions/catalog-view-properties-side-panel
so it's not an internal implementation.